pkg/api/division.go

```diff
@@ -0,0 +1,95 @@
+package api
+
+import (
+	"encoding/json"
+	"html/template"
+	"log"
+	"net/http"
+
+	"github.com/jeka2708/golang-training-enterprise/pkg/data"
+)
+
+type pageDataDivision struct {
+	Title     string
+	Divisions []data.Division
+}
+
+func (a dataAPI) getAllDivision(writer http.ResponseWriter, request *http.Request) {
+	dv, err := a.data.ReadAllDivision()
+
+	pD := pageDataDivision{
+		Title:     "Список отделов",
+		Divisions: dv,
+	}
+	if err != nil {
+		_, err := writer.Write([]byte("got an error when tried to get divisions"))
+		if err != nil {
+			log.Println(err)
+		}
+	}
```

Comment:
You just printing an error, and not exit from function, so in step: encoding go model to json you will get an error again.
So please add writer.WriteHeader(http.StatusInternalServerError) and exit of function after printing log message.